### DISCUSSION TITLE
Latest release doesn't show up on the update screen.

### DISCUSSION THREAD

**Discussion by Rjaap** [rern/rAudio#1401] [Bug Reports]
Currently on 20230630, on a RPI 3B+. Never had a problem updating until now. Github says the latest release is 20230824 but it simply will not show up on the update screen and the button itself stays black.

Already tried, without result:
- reboot 
- update (sudo pacman -Syu, no problems), then reboot

Any ideas?

**Comment by klassikmann** [comment]
@Rjaap 
As long as addonslist.json in https://github.com/rern/rAudio-addons isn't updated the release 20230824 will not show up.

**Reply by Rjaap** [reply]
Ah ok. I'll keep that in mind next time github warns me about a new release. Thanks!

**Reply by klassikmann** [reply]
@Rjaap 
addonslist.json has been updated, therefore you are able to update rAudio to 20230828.

**Reply by Rjaap** [reply]
Works. Thanks for the heads up!
